py-tensorrt-text-detection/README.md

```diff
@@ -0,0 +1,112 @@
+
+#### Method 2 (Recommended): Use [tensorRT container](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tensorrt)
+1. Install Nvidia docker tool chain, following the guide [on this page](https://github.com/NVIDIA/libnvidia-container)
+1. You may run into Nvidia docker problems, check if this [link](https://github.com/NVIDIA/nvidia-docker/issues/1447) helps or this [link](https://bbs.archlinux.org/viewtopic.php?id=266915)
+1. (MUST) Please run the following command before installing opencv python packages:
+```bash
+apt-get update
+apt-get install ffmpeg libsm6 libxext6  -y
+```
+### Install Dependencies
+Run the following command:
+```
+pip install -r ../requirements.txt
+pip install -r requirements.txt
+```
+## Model Conversion (ONNX to TensorRT)
+
+*We don't provide sample for INT8 conversion, you need to provide a calibration file, please follow the official guidelines.*
+### Method 1: Using `polygraphy` or `trexec`
+This requires trexec being compiled and installed on the system (check [TensorRT git repo](https://github.com/NVIDIA/TensorRT/tree/main/samples/trtexec) for installation guide), 
+or if you are using TensorRT container, it is included.
+
+```bash
+polygraphy convert dynamic_identity.onnx -o dynamic_identity.engine \
+    --trt-min-shapes X:[1,3,28,28] --trt-opt-shapes X:[1,3,28,28] --trt-max-shapes X:[1,3,28,28] \
+```
+After converting, you can inspect the converted engine with polygraphy to make sure the converted engine is correct. Note that, please set the min, opt and max shape correspondingly based on your application scenario.
+
+### Method 2: Using tensorRT libraries (see utility file `trt_utils.py`)
+We only demonstrate the conversion using python API, in the function `_build_engine_onnx(self) -> trt.tensorrt.ICudaEngine`.
+
+## Related Files for this folder
+The following files are required to run the `tensorrt_notebook.ipynb` notebook.
+Please refer to the parent level readme for details of each file.
+
+```bash
+├── model_common
+│   ├── __init__.py
+│   ├── paddle_decoder.py
+│   ├── paddle_operators.py
+│   ├── paddle_preprocess.py
+│   └── utils.py
+├── models
+│   └── ch_PP-OCRv2_det_infer.onnx
+├── py-tensorrt-text-detection
+│   ├── README.md
+│   ├── requirements.txt
+│   ├── tensorrt_notebook.ipynb
+│   └── trt_utils.py
+├── requirements.txt
+└── sample_images
+    └── nutrition_label.jpg
+```
+- `trt_utils.py` is the python script showing how to convert an onnx model to tensorRT engine using python API and then run single inference using the converted tensorRT engine. Note that, the onnx to tensorRT engine conversion is tailored to paddlepaddle text detection model,  PLEASE MODIFY IT ACCORDINGLY BASED ON YOUR MODEL.
+- `tensorrt_notebook.ipynb` is a python notebook that demostrates the model conversion and inference process using tensorRT engine and PaddlePaddle text detection model.
```

Comment:
demonstrate

Reply:
a notebook demonstrates 

isn't it correct?